selenoid_test.go

```diff
@@ -980,6 +981,75 @@ func TestAddedSeCdpCapability(t *testing.T) {
 	queue.Release()
 }
 
+func TestBidi(t *testing.T) {
+	manager = &HTTPTest{Handler: Selenium()}
+
+	resp, err := http.Post(With(srv.URL).Path("/wd/hub/session"), "", bytes.NewReader([]byte("{}")))
+	assert.NoError(t, err)
+	assert.Equal(t, resp.StatusCode, http.StatusOK)
+	var sess map[string]string
+	assert.NoError(t, json.NewDecoder(resp.Body).Decode(&sess))
+
+	u := fmt.Sprintf("ws://%s/session/%s", srv.Listener.Addr().String(), sess["sessionId"])
+
+	ctx, cancel := context.WithTimeout(context.Background(), 200*time.Millisecond)
+	defer cancel()
+
+	dialer := websocket.Dialer{}
```

Comment:
Here I use ws library to check bidi (in devtools used `github.com/mafredri/cdp/rpcc`